### PR TITLE
Remove questions that don't make sense in this repo.

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made?
This repo is a test.  Typically we don't test our tests.

## How was this change tested?

It wasn't

## Which documentation and/or configurations were updated?
none

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,2 @@
 ## Why was this change made?
```
Comment (justinlittman) [inline .github/pull_request_template.md:2]: The question about docs/configs still makes sense.

Comment (jcoyne) [reply .github/pull_request_template.md:2]: All the documentation is in the repo.  So asking "which" is odd.  Clearly if it's in the diff, then it's been updated. right?

For the configuration question should we be specific and say "shared configs"?  That is the only configuration outside of this repository.

Comment (justinlittman) [reply .github/pull_request_template.md:2]: From my perspective, the goal is to remind the developer and reviewer to write/review doc and config changes. That is accomplished by the words "documentation" and "configuration". Other than that, not too concerned with the wording.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -1,12 +1,2 @@
...
 
-
-
-## How was this change tested?
-
-
-
-## Which documentation and/or configurations were updated?
```
Comment (jcoyne) [inline .github/pull_request_template.md:9]:
```suggestion
## Was README.md updated

## Are there any changes required in shared_configs
```

**Pull request merged**